locales-src/newtab.ftl

```diff
@@ -83,6 +83,13 @@ newtab-menu-delete-history = Delete from History
 newtab-menu-save-to-pocket = Save to { -pocket-brand-name }
 newtab-menu-delete-pocket = Delete from { -pocket-brand-name }
 newtab-menu-archive-pocket = Archive in { -pocket-brand-name }
+newtab-menu-show-privacy-info = Our sponsors & your privacy
+
+## newtab privacy info modal
```

Comment:
Leave an empty line after a section comment. Also, these comments are supposed to be read by localizers, can it be a bit more explanatory?